jupyter_server/serverapp.py

```diff
@@ -546,6 +569,25 @@ class ServerApp(JupyterApp):
         password=(JupyterPasswordApp, JupyterPasswordApp.description.splitlines()[0]),
     )
 
+    # A list of service handlers to expose.
+    # Subclasses can override this list to
+    # expose a subset of these handlers.
+    exposed_services = [
```

Comment:
I'm really sorry Zach.  I hadn't realized that `extra_services` existed until now.  In light of that, I think `default_services` is probably a better choice than `exposed_services` since I figured this was the complete set.  (One of my follow-on questions was how one goes about overriding the handlers for an existing service.  Now I know. :blush:)

If you do choose to retain `exposed_services`, you'll want to update the reference to `self.default_services` in the [`ServerWebApplication` constructor](https://github.com/jupyter/jupyter_server/pull/84/files#diff-8a07fc4bce4a1b879d161c56f1524b2cR1265) and the corresponding methods.  Perhaps that was a sign that `default_services` was better in the first place.  My apologies.

Reply:
No problem! `extra_services` wasn't highlighted in this PR, so I wouldn't expect anyone to notice it. I didn't know it exists until working on this PR 😆.  

`exposed_services` versus `default_services`...I don't have a strong preference either way. We can stick with default services if that makes sense to you. I'll fix the name mismatch in the webapp.